docs/pages/openssh-teleport.mdx

```diff
@@ -1,352 +0,0 @@
----
-title: Using Teleport with OpenSSH
-description: How to use Teleport on legacy systems with OpenSSH and sshd
----
-
-Teleport is fully compatible with OpenSSH and can be quickly setup to record and
-audit all SSH activity. Using Teleport and OpenSSH has the advantage of getting you up
-and running, but in the long run we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://gravitational.com/blog/openssh-vs-teleport/)
```

Comment:
What happened with this page? I can't find the new OpenSSH page?

Reply:
It got moved a long time ago to: https://goteleport.com/docs/server-access/guides/openssh/ - the redirect should also go to that page. I think, whoever merged 6.1 into master ended up not doing this correctly, certain rebasing strategies will cause files that were removed to get merged _back_ in?

Reply:
Good to know, thanks for the clarification.